src/catkin_pkg/package_templates.py

```diff
@@ -205,10 +206,10 @@ def create_package_files(target_path, package_template, rosdistro,
     manifest_path = os.path.join(target_path, PACKAGE_MANIFEST_FILENAME)
     if manifest_path not in newfiles:
         newfiles[manifest_path] = \
-            create_package_xml(package_template, rosdistro)
+            create_package_xml(package_template, rosdistro, meta)
```

Comment:
I personally don't like the **kwargs solution I managed to implement, I preferred the previous one for the meta argument. That's why I'm keeping the history of the previous commit as well.

The PR is updated now.